app/controllers/users.js

```diff
@@ -82,37 +88,111 @@ exports.checkAvatar = function (req, res) {
 /**
  * Create user
  */
-exports.create = function (req, res) {
-  if (req.body.name && req.body.password && req.body.email) {
+// exports.create = (req, res) => {
+//   if (req.body.name &&
+//     req.body.password &&
+//     req.body.email) {
+//     User.findOne({
+//       email: req.body.email
+//     }).exec((err, existingUser) => {
+//       if (!existingUser) {
+//         const user = new User(req.body);
+//         // Switch the user's avatar index to an actual avatar url
+//         user.avatar = avatars[user.avatar];
+//         user.provider = 'local';
+//         user.save((err) => {
+//           if (err) {
+//             return res.render('/#!/signup?error=unknown', {
+//               errors: err.errors,
+//               user
+//             });
+//           }
+//           req.logIn(user, (err) => {
+//             if (err) return next(err);
+//             return res.redirect('/#!/');
+//           });
+//         });
+//       } else {
+//         return res.redirect('/#!/signup?error=existinguser');
+//       }
+//     });
+//   } else {
+//     return res.redirect('/#!/signup?error=incomplete');
+//   }
+// };
```

Comment:
What is this code still doing? If it is not doing anything, you can feel free to remove it. There is no point using ghost code in our project